explainers/navigation-to-unsigned-bundles.md

```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
   removing query parameters, but if a significant number of users cleaned their
   URLs, the tracking could move to path segments.
 
+* The path of a locally-saved bundle may include private information like a
+  username. As [Loading an untrusted bundle](#loading-an-untrusted-bundle)
+  suggests, the URL of the bundle itself needs to be hidden from web APIs to
+  avoid exposing this.
+
+### Security/Privacy Questionaire
+
+This section contains answers to the [W3C TAG Security and Privacy
+Questionnaire](https://w3ctag.github.io/security-questionnaire/).
+
+#### 1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature should not expose additional information than what the original Web
+sites intended to expose.
+
+#### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes.
+
+#### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+When the content included in the unsigned Web Bundle is loaded as an untrusted
+bundle (e.g: loaded a “untrusted” bundle from a local file), navigation to the
+Bundle will be loaded within its unique origin with package: scheme, which is
+different from the claimed origin or from other bundles, therefore no access to
+the original origin’s storage should be given.
+
+Please also see the [Security and privacy considerations](#security-and-privacy-considerations).
+
+#### 4. How does this specification deal with sensitive information?
+
+(See above) TODO: link.
+
+#### 5. Does this specification introduce a new state for an origin that persists across browsing sessions?
+
+This proposal introduces a [`package:` scheme](#urls-for-bundle-components) that
+defines an origin based on the location or the URL of the bundle itself, and the
+claimed URL inside the bundle. That's a new kind of origin with the same state
+as any other origin (e.g. indexed DB, Local Storage), which can be persisted
+across browsing sessions.
+
+#### 6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+None.
+
+#### 7. Does this specification allow an origin access to sensors on a user’s device
+
+No.
+
+#### 8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+Network requests (e.g. fetch or iframe) from the unsigned bundle could expose IP
+address of the user, in the same way as regular navigation does.  See also this
```

Comment:
https://github.com/WICG/webpackage/blob/master/explainers/navigation-to-unsigned-bundles.md#network-access